core/invehicle-digital-twin/src/main.rs

```diff
@@ -2,6 +2,8 @@
 // Licensed under the MIT license.
 // SPDX-License-Identifier: MIT
 
+use common::grpc_interceptor::GrpcInterceptorLayer;
+use common::grpc_interceptor_samples::sample_grpc_interceptor_factory;
```

Comment:
why is sample code being used in the core services?

Reply:
This sample has to be part of core, as it is coupled with the code in core/common.  It will soon be removed from the main.rs code.

Reply:
Removed and placed into the documentation.